src/Test.Utilities/WorkItemAttribute.cs

```diff
@@ -7,13 +7,10 @@ namespace Test.Utilities
     [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class)]
     public class WorkItemAttribute : Attribute
     {
-        private readonly int _id;
-        private readonly string _source;
-
+#pragma warning disable IDE0060 // Remove unused parameter
+#pragma warning disable CA1801 // Remove unused parameter
```

Comment:
Probably just change fields to properties instead?

Reply:
We will still get diagnostics about unused properties (if they are private). Alternatively, I can make these public properties. Let me do the change.

Reply:
Actually, will do as a follow up change. I am working on a fork of this branch to harness our CodeStyle (formatting analyzer) NuGet package for the repo.